launch_scripts/train_captioner.py

```diff
@@ -123,9 +103,9 @@
             persistent_workers=True,
         ),
     )
+    duration_factor = 4
```

Comment:
Is this correct? It seems like it will divide the warmups by 4

Reply:
ahh, good catch, forgot to reverse this change, will fix!